### DISCUSSION TITLE
这是停更了么？还是说只有重大版本才会更新了？

### DISCUSSION THREAD

**Discussion by VincentShin** [osy86/HaC-Mini#913] [General]
如题

**Comment by controlfix87** [comment]
据我所知，这个项目正在维护...... :)

**Reply by VincentShin** [reply]
原来如此，谢谢🙏！

**Comment by Medizinmann-MD** [comment]
See also - Release Schedule....
https://github.com/osy/HaC-Mini/releases

osy switched from monthly updates to only when necessary and maybe major releases for new MacOS Versions i.e. Monterey(10.12) if necessary.

**Reply by VincentShin** [reply]
I got it. Thanks, bro
